.github/workflows/rust.yml

```diff
@@ -18,6 +18,6 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Build
-      run: cargo build --verbose
+      run: cargo build --verbose --features=v2021_3
```

Comment:
I think this can catch a broken build, but not a broken test (like the one that happened this time). The `cargo test` call below likely needs a recent version too.

Reply:
Good catch!  Fixed.